Prüfschritte/de/11.5.2.13 Fokusverfolgung Texteinfügemarke und Textauswahl.adoc

```diff
@@ -9,17 +9,15 @@ Dieser Prüfschritt basiert auf den Kriterien der EN 301 549 Abschnitt
 
 == Was wird geprüft?
 
-Informationen und Mechanismen für die Fokusverfolgung, Einfügemarke und
+Informationen und Mechanismen für die Fokusverfolgung, Texteinfügemarke und
 Textauswahl müssen für die Hilfsmitteltechnologie programatisch verfügbar
 sein.
 
 == Wie wird geprüft?
 
-* Kann Text mittels Screenreader navigiert werden?
-* Kann die Einfügemarke bzw. der Cursor versetzt werden und ist dies mit dem
-  Screenreader nachvollziehbar?
-* Kann Text ausgewählt werden und ist dies auch mittels Hilfstechnologie
-  möglich?
+* Wird die Fokusposition vom Screenreader ausgegeben?
+* Wird die Position der Texteinfügemarke vom Screenreader ausgegeben?
```

Comment:
Ich denke, die Beschreibung ist etwas missverständlich. So wird bei incrementeller Auswahl in OS z.B. nicht immer der ganze zuvor ausgewählte Text ausgegeben, sondern der zusätzlich ausgewählte. Bei verringerung der Answahl wird der nun nicht mehr ausgewählte Bestandteil mit Zusatz "nicht ausgewählt" ausgegeben, als gerade nicht der ausgewählte Text. Da sich die Mechanismen der Auswahl und der Vermittlung von ausgewäghlten und nicht ausgewähltem Text je nach Screenreader und OS und auch zukünftig unterscheiden können, finde ich eine generellere Prüfung im Sinne von "Unterstützt der Inlat, soweit verfügbar, die vom System und Screenreader vorgesehenen mechanismen der Textauswahl?" Das scheint mir in der bestehenden Prüfanleitung besser umgesätzt als in dieser Umformulierung.